docs/iris/src/developers_guide/release.rst

```diff
@@ -0,0 +1,72 @@
+Releases
+********
+
+A release of Iris is a tag on github in the SciTools organisation master.
```

Comment:
Interesting sentence. Perhaps this is improved by:

```
A release of Iris is a tag on the SciTools/iris repository.
```

Or is there meaning that I've missed in the sentence?